tutorials/object_gcr_4_photoz.ipynb

```diff
@@ -0,0 +1,271 @@
+    "\n",
+    "__Learning objectives__: After going through this notebook, you should be able to:\n",
+    "  1. Load and efficiently access a DC2 object catalog (+ photo-z) with the GCR\n",
+    "  2. Understand how the photo-z data are stored / represented\n",
+    "  3. Look at an example of galaxy photo-z distributions\n",
+    "  \n",
+    "__Logistics__: This notebook is intended to be run through the JupyterHub NERSC interface available here: https://jupyter-dev.nersc.gov. To setup your NERSC environment, please follow the instructions available here: https://confluence.slac.stanford.edu/display/LSSTDESC/Using+Jupyter-dev+at+NERSC\n",
+    "\n",
+    "__Other notes__: \n",
+    "If you restart your kernel, or if it automatically restarts for some reason, all imports and variables will become undefined so, you will have to re-run everything."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0, '../../gcr-catalogs/')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import GCRCatalogs\n",
+    "from GCR import GCRQuery"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the catalog\n",
+    "\n",
+    "Loading the object catalog with photo-z add-on. The catalog name is `dc2_object_run1.2i_with_photoz`. \n",
+    "\n",
+    "**Note**: if you need more quantities (including those not in DPDD), then use `dc2_object_run1.2i_all_columns_with_photoz` instead.\n",
+    "\n",
+    "It takes a few seconds for the catalog instance to initiate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cat = GCRCatalogs.load_catalog('dc2_object_run1.2i_with_photoz')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Photo-z access methods\n",
+    "\n",
+    "The photo-z infomation can be accessed in two ways:\n",
+    "\n",
+    "1. As a single column `photoz_mode` that has the mode value (z_peak), and\n",
+    "2. As a single multi-dimension coulmn `photoz_pdf` (i.e., when accessing this column, you get a 2D array instead of a 1D one). \n",
+    "\n",
+    "We will demostrate both access method in detail. You can notice that all the photo-z columns have a prefix of `photoz_`. \n",
+    "\n",
+    "Let's first make sure that these columns are indeed available. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sorted(q for q in cat.list_all_quantities() if q.startswith('photoz_'))"
```

Comment:
Somewhere near the top of the notebook, perhaps around this point, I would add an explicit mention of pz_pdf_bin_centers as the points at which the PDF was evaluated.